validation_run.py

```diff
@@ -0,0 +1,182 @@
+DESIGN_ACH = [3, 8]
+SPEAKING_PCT = np.asarray([0.55, 0.66])
+SPEAKING_VOL = 70
+NUMBER_DAYS = 14
+PROBABILITY_CONTACT_REMAIN = 0.4
+EMPIRICAL = 76
+
+# ## San Diego VA Office #
+# print("*"*5, 'San Diego VA office', '*'*5)
+# EMPLOYEE=100
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.4]
+# NUMBER_DAYS=8
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=5
+
+# ## singapore conference
+# print("*"*5, 'Singapore conference', '*'*5)
+# EMPLOYEE=111
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[6,12]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.25,.4]
+# NUMBER_DAYS=3
+# PROBABILITY_CONTACT_REMAIN=0
+# EMPIRICAL=7
+
+# ## SD meat processing Shift 1
+# print("*"*5, 'SD Meat Processing Shfit 1', '*'*5)
+# EMPLOYEE=1744
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[2,22]
+# SPEAKING_PCT=[.05,.2]
+# NUMBER_DAYS=14
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=32
+
+# ## SD meat processing shift 2
+# print("*"*5, 'SD Meat Processing Shfit 2', '*'*5)
+# EMPLOYEE=1459
+# WFH=0
+# CASE_RATE=1/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[2,22]
+# SPEAKING_PCT=[.05,.2]
+# NUMBER_DAYS=14
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=6
+
+# ## Henan expressway
+# print("*"*5, 'Henan Expressway', '*'*5)
+# ## to run this case, skip the block below to calculate SAR,  instead enter SAR = [0.2, 0.2]
+# EMPLOYEE=103
+# WFH=0
+# CASE_RATE=1/100000
+# DAILY_AVERAGE_CONTACTS=[2,4]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=13
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=6
+
+# ## MLB with mask protocol and social distancing
+# print("*"*5, 'MLB with mask and social distancing', '*'*5)
+# EMPLOYEE=68
+# WFH=0
+# CASE_RATE=60/1e5
+# DAILY_AVERAGE_CONTACTS=[4,8]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=20
+
+# ## MLB without mask protocol and social distancing
+# print("*"*5, 'MLB without mask and social distancing', '*'*5)
+# EMPLOYEE=68
+# WFH=0
+# CASE_RATE=60/1e5
+# DAILY_AVERAGE_CONTACTS=[4,10]
+# DESIGN_ACH=[3,8]
+# SPEAKING_PCT=[.05,.25]
+# NUMBER_DAYS=10
+# PROBABILITY_CONTACT_REMAIN=.4
+# EMPIRICAL=20
+
+######## simulations ##########
+######## Please note that this takes significant amount of time ############
+
+## calculate SAR
+## skip this for Henan Expressway Outbreak
+## Henan Expressway SAR = np.asarray([0.2, 0.2])
+SAR = np.asarray([])
+for pct in SPEAKING_PCT:
+    SAR = np.append(
+        SAR,
+        new_model.get_sar_from_speaking_airflow(
+            design_airflow_ACH=DESIGN_ACH,
+            filtration_per_hour=0,
+            speaking_percentage=pct,
+            speaking_volume=SPEAKING_VOL,
+            mask_effectiveness=0.0,
+        ),
+    )
+
+## calculate the number of simulation required
+## this follows negative binomial distribution
+VALID_SIM = 5000
+df = np.random.negative_binomial(VALID_SIM / CASE_RATE, CASE_RATE, VALID_SIM)
```

Comment:
This statement is not clear to me. What's the purpose of negative binomial here?

Reply:
This is a mistake. I was thinking the # of trials required to achieve X number of success is a negative binomial. Here we are doing multiple Bernoulli trials each day instead of only one trial.